src/api/store/community.py

```diff
@@ -212,21 +212,19 @@ def _update_locations(self, geography_type, locations, community):
                     zipcode = zipcodes.matching(location)
                     if len(zipcode) > 0:
                         city = zipcode[0].get("city", None)
+                        county  = zipcode[0].get("county", None)
+                        state = zipcode[0].get("state", None)
                     else:
                         raise Exception("No zip code entry found for zip=" + location)
 
                     # get_or_create gives an error if multiple such locations exist (which can happen)
                     # loc, created = Location.objects.get_or_create(location_type='ZIP_CODE_ONLY', zipcode=location, city=city)
-                    loc = Location.objects.filter(
-                        location_type="ZIP_CODE_ONLY", zipcode=location, city=city
+                    loc, created = Location.objects.get_or_create(
+                        location_type="FULL_ADDRESS", zipcode=location, city=city, county=county, state=state
                     )
-                    if not loc:
-                        loc = Location.objects.create(
-                            location_type="ZIP_CODE_ONLY", zipcode=location, city=city
-                        )
+                    if created:
                         print("Zipcode " + location + " created for town " + city)
                     else:
-                        loc = loc.first()
                         print("Zipcode " + location + " found for town " + city)
```

Comment:
@BradHN1, is there a reason for the print statements?

Reply:
originally for testing.  removing most of them